### ISSUE TITLE
Theme Updates

### ISSUE THREAD

**Issue by greenervigil** [greenervigil/next-portfolio#60]
A11y fails on the color contrast of the white text against the teal background.  Make the following selected changes to adhere to both AA and AAA Guidelines.

teal-500
neutral-800 - large text
neutral-900 - smaller text

<img width="648" alt="image" src="https://user-images.githubusercontent.com/16470877/221481371-b43255b8-9ebf-4312-b73c-2c29043ecec0.png">
